package.json

```diff
@@ -30,11 +30,9 @@
     "test": "jest",
     "typescript": "tsc --noEmit",
     "lint": "eslint \"**/*.{js,ts,tsx}\"",
-    "prepare": "bob build",
```

Comment:
Is `bob build` not required or is removing this intentional and it has always been a redundant step during the build/compile ?

IMO, [npm prepare](https://docs.npmjs.com/cli/v9/using-npm/scripts) command is run before `npm install`

Disclaimer: I'm not fully aware of its' purpose in the current build.

Reply:
@vharsh   Yes, We are removing this intentionally Since this is causing issues in windows machine while adding tuvali dependency to inji

Reply:
Looks good if we are able to build it on Mac as well.